Coroutine.ttslua

```diff
@@ -1,16 +1,28 @@
 ---@class ge_tts__Coroutine
 local Coroutine = {}
 
---- Yields from the current coroutine, which will later be resumed once a condition is met.
+--- Yields from the current coroutine, to be resumed once a condition is met or an optional timeout limit is reached.
 ---@param condition fun(): boolean @Return true when the current coroutine should be resumed.
-function Coroutine.yieldCondition(condition)
+---@param timeout number|nil @Timeout in seconds (optional).
+---@return boolean @Return whether the condition was met or not.
```

Comment:
Seems as this is documenting the return value, there's no need to include "Return" in the description.

Additionally, the language "Whether or not" does not explain the "not" case. I'd suggest:

> true if the condition was met, or false if the (optional) timeout was reached.